### PR TITLE
Request to merge first sprint to master branch

### PR DESCRIPTION
**Pull request by tgqiang**
Full Team code review will be done for this first merge. Subsequently, future pull requests will only require 2 code reviewers.

### CODE REVIEW

File: components/model.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+## Connects to the postgres database
+connection = components.database_adapter.connect_db()
+db_cursor = connection.cursor()
+
+## Get the module code, name, description, and MCs of all modules
+def getAllModules():
+	sql_command = "SELECT * FROM module ORDER BY code"
+	db_cursor.execute(sql_command)
+	return db_cursor.fetchall()
+
```
Comment (tgqiang) [inline components/model.py]: Team-agreed convention to use 2 newlines after each function definition, to add 1 more newline.

File: components/model.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+## Get the module code, name, description and MCs of a single module
+def getModule(code):
+	sql_command = "SELECT * FROM module WHERE code=%s"
+	db_cursor.execute(sql_command, (code,))
+	return db_cursor.fetchone()
+
+## Get the module code, name, AY/Sem and quota of all fixed mounted modules
+def getAllFixedMountedModules():
+	sql_command = "SELECT m2.moduleCode, m1.name, m2.acadYearAndSem, m2.quota FROM module m1, moduleMounted m2 WHERE m2.moduleCode = m1.code ORDER BY m2.moduleCode, m2.acadYearAndSem"
```
Comment (tgqiang) [inline components/model.py]: This is too long.

File: templates/base.html [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+		<script type="text/javascript" src="static/javascripts/movetop.js"></script>
+	</head>
+
+	<body>
+		<button class="btn btn-primary" id="movebtn" onclick="moveToTop()"><span class="glyphicon glyphicon-triangle-top"></span> Back to Top</button>
+
+		<nav class="navbar navbar-default">
+			<div class="container-fluid">
+			    <form class="navbar-form navbar-left">
+				    <button type="button" class="btn btn-primary" onclick="openSidebar()">
```
Comment (tgqiang) [inline templates/base.html]: Indentations are off, please fix.

File: test/test_database.py [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+        self.mountingTableColumnCount = 3
+        self.testModuleMountedCount = 2
+        self.testModuleFixedMountedAySem1 = "AY 16/17 Sem 1"
+        self.testModuleFixedMountedAySem2 = "AY 16/17 Sem 2"
+        self.testModuleTentativeMountedAySem1 = "AY 17/18 Sem 1"
+        self.testModuleTentativeMountedAySem2 = "AY 17/18 Sem 2"
+        self.testModuleQuota1 = 190
+        self.testModuleQuota2 = 160
+    
+    def test_getModule(self):
```
Comment (tgqiang) [inline test/test_database.py]: Function naming convention needs to be changed to test_<something>_<something>...

File: components/database_adapter.py [hunk 5]
```diff
@@ -0,0 +1,75 @@
+''' database_adapter.py handles the connection to the database. '''
```
Comment (helloqx) [inline components/database_adapter.py]: Team-agreed convention to start comment on a newline after '''

**Pull request merged**